src/components/CommentForm.jsx

```diff
@@ -0,0 +1,92 @@
+import { useState } from "react";
+import { addComment } from "../api";
+
+export const CommentForm = ({ articleId, comments, setComments }) => {
+  const [usernameInput, setUsernameInput] = useState("");
+  const [commentInput, setCommentInput] = useState("");
+  const [isSubmitted, setIsSubmitted] = useState(false);
+  const [error, setError] = useState(null);
+  const [usernameErrorMsg, setUsernameErrorMsg] = useState(null);
+  const [commentErrorMsg, setCommentErrorMsg] = useState(null);
+
+  const handleUsernameBlur = () => {
+    if (usernameInput.length < 2) {
+      setUsernameErrorMsg('Username must be longer than 1 character.');
+    } else {
+      setUsernameErrorMsg('');
+    }
+  };
```

Comment:
Instead of having a user have to type a username, may be easier to let then select one from a drop down to avoid an incorrect username being used